tests/management/group/test_view.py

```diff
@@ -2914,6 +2914,22 @@ def test_get_group_service_account_username_only_success(self):
         for sa in response.data.get("data"):
             self.assertIn(sa["username"], expected_usernames)
 
+    def test_get_group_principal_type_all_username_only_success(self):
```

Comment:
this endpoint does not support `type` query param so please either remove the test or update it to match an existing scenario